{"uuid":"87e6d040-5e9b-465a-90f0-10fc2a080997","name":"Scenario: User enters Invalid Python code and Presented with Error Message","children":["fe8cddcf-2d6a-4411-807f-692bdb759af0"],"befores":[],"afters":[{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687876760959,"stop":1687876760959},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687876761179,"stop":1687876761180},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687876761329,"stop":1687876761329},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687876761406,"stop":1687876761406},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687876761423,"stop":1687876761424}],"start":1687876760847,"stop":1687876761424}